feat: edits to catalogue model

BrendaHijmans opened the pull request:
What are the main changes you did:
-Edits to descriptions in several tables.
- Variable values model update

- Edit Resource model
   - Add Resources.dateLastRefresh
   - Delete Resources.dateEstablished
   - Rename Resources.startDataCollection to  Resources.endYear
   
- Change order of some attributes in Resources table

how to test:
- explain here what to do to test this (or point to unit tests)

todo:
- [ ] updated docs in case of new feature
- [ ] added/updated tests
- [ ] added/updated testplan to include a test for this fix, including ref to bug using # notation

mswertz commented on data/_models/shared/Contacts.csv:
```diff
@@ -1,19 +1,19 @@
+Contacts,,statement of consent personal data,bool,,TRUE,,,,,,,"Indicate whether this person consents to display the following personal data in the catalogue (which is openly accessible): title, name, photo, orcid ID, institute, department, expertise, discipline and role within the resource. These data-items may partly be extracted from the UMCG and/or RUG employee pages. The consent also includes ...
+Contacts,,prefix,,,,,,,,,,"Surname prefix, if applicable","DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging",,
+Contacts,,initials,,,,,,,,,,Initials of the contact person,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging",,
+Contacts,,title,ontology,,,CatalogueOntologies,Titles,,,,foaf:title,Title of the contact person,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging",,
```
you want to do the 'pre' and 'post' titles later?

BrendaHijmans replied:
They all go under 'title' I think? 'prefix' is meant for e.g. 'van der' in 'van der Velde'

mswertz replied:
ah no. You have titles before, such as 'Prof.dr.' and there are pre-nominal letters such as 'PhD'
see https://rapidbi.com/post-nominal-letters-and-qualifications/
but I don't mind to revisit this later.

BrendaHijmans replied:
Yes I understood. I would like to do this later, it is not within the scope of the migrations.

hslh replied:
I made a separate issue for this here: https://github.com/molgenis/GCC/issues/716

BrendaHijmans replied:
great :)

mswertz reviewed (approved):
I notice we really should create a new column type that understands 'other' and then allows custom terms to be added to the list when 'other' is selected (like a special flavor of ontology)

BrendaHijmans commented:
> The model is updated ( fields are renamed, removed and added ) therefore the version should be bumped to 4.1 and migrations steps/scripts should be added

I reckoned this was still part of setting up data model 4.0, since it is not in production yet. The migrations to 4.0 are in a different pull request, it is not practical to have them in this pull request. Also, we will not migrate from 3.x to 4.0 and then again to 4.1
